packages/wp-story-editor/src/api/media.js

```diff
@@ -100,8 +104,8 @@ export async function getMutedMediaById(config, mediaId) {
 
   const result = await apiFetch({ path });
 
-  if (result?.meta?.web_stories_muted_id) {
-    return getMediaById(config, result.meta.web_stories_muted_id);
+  if (result?.meta?.webStoriesMutedId) {
```

Comment:
I don't think the result we get from the REST API here is camel-cased already.

Reply:
Sorry, my bad. Done in https://github.com/google/web-stories-wp/pull/9683/commits/c29543c002a5f6c407e0d8f56b2d878b44ae7acf